setup.cfg

```diff
@@ -58,9 +58,10 @@ test =
   pytest-ordering
   pytest-cov
 docs =
-  sphinx
+  sphinx<7.0.0  
   sphinx_rtd_theme
   m2r2
+# pin sphinx because 7.0 currently not compatible with rtd theme
```

Comment:
Looks like RTD is now: `sphinx >=5,<8` so we could try to upgrade Sphinx at some point. Not a blocker for this though